### DISCUSSION TITLE
您好 我想咨询下  您这个目前支持post方法了么

### DISCUSSION THREAD

**Discussion by qiubeixiansheng** [chenshuo/muduo#645] [General]
DELETE FROM HERE TO LINE 18 BEFORE OPEN AN ISSUE.

GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, use GitHub discussions:

https://github.com/chenshuo/muduo/discussions

When filing an issue of muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

**Comment by chenshuo** [comment]
Nope.
